pkg/apis/authentication/v1alpha1/types_adminkubeconfig_request.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// AdminKubeConfigRequest is a OpenID Connect configuration that is applied
+// to a Shoot in a namespace.
+type AdminKubeConfigRequest struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+	// Spec is the specification of this OpenIDConnect preset.
+	Spec AdminKubeConfigRequestSpec `json:"spec" protobuf:"bytes,2,opt,name=spec"`
+	// Spec is the specification of this OpenIDConnect preset.
+	Status AdminKubeConfigRequestStatus `json:"status" protobuf:"bytes,3,opt,name=status"`
+}
+
+type AdminKubeConfigRequestStatus struct {
+	KubeConfig []byte `json:"kubeConfig,omitempty" protobuf:"bytes,1,opt,name=kubeConfig"`
```

Comment:
```suggestion
	Kubeconfig []byte `json:"kubeconfig,omitempty" protobuf:"bytes,1,opt,name=kubeconfig"`
```

Again, let's keep this similar to other resources that contain kubeconfigs